rpc_jobs/rpc_aio.yml

```diff
@@ -225,8 +226,10 @@
             }} catch (e) {{
               print(e)
               throw e
+              currentBuild.result = 'FAILURE'
```

Comment:
This is before the exception is thrown. Its in the catch block so that it only runs on failure.